common/src/main/scala/org/apache/celeborn/common/util/PbSerDeUtils.scala

```diff
@@ -278,6 +278,12 @@ object PbSerDeUtils {
     if (masterPersistWorkerNetworkLocation) {
       workerInfo.networkLocation = pbWorkerInfo.getNetworkLocation
     }
+    // If the next interruption notice is not specified in the message, set to max.
+    if (pbWorkerInfo.getNextInterruptionNotice == 0) {
+      workerInfo.nextInterruptionNotice_$eq(Long.MaxValue)
+    } else {
+      workerInfo.nextInterruptionNotice_$eq(pbWorkerInfo.getNextInterruptionNotice)
```

Comment:
```suggestion
      workerInfo.nextInterruptionNotice = pbWorkerInfo.getNextInterruptionNotice
```

Reply:
PbSerDeUtils is a Scala class, which means that you don't need to use the equals method.